app/utils/extract-doi.ts

```diff
@@ -0,0 +1,11 @@
+// https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/utils/extract-doi-from-string.js
```

Comment:
I understand not wanting to be held responsible for this regex, but better than linking ember-osf would be adding some unit tests!